File: jobs/2026-06-16__11-15-58/file_listing_cursor_aggregation__8RcYdti/artifacts/user/apideck_task/check.js
const https = require('https');

async function request(url, options, body) {
  return new Promise((resolve, reject) => {
    const req = https.request(url, options, (res) => {
      let data = '';
      res.on('data', chunk => data += chunk);
      res.on('end', () => resolve({ status: res.statusCode, data: JSON.parse(data) }));
    });
    req.on('error', reject);
    if (body) {
      req.write(body);
    }
    req.end();
  });
}

(async () => {
  const headers = {
    'Authorization': `Bearer ${process.env.APIDECK_API_KEY}`,
    'x-apideck-app-id': process.env.APIDECK_APP_ID,
    'x-apideck-consumer-id': process.env.APIDECK_CONSUMER_ID,
    'x-apideck-service-id': 'onedrive'
  };

  const res = await request('https://unify.apideck.com/file-storage/drives', { headers });
  console.log(JSON.stringify(res.data, null, 2));
})();

File: jobs/2026-06-16__11-15-58/file_listing_cursor_aggregation__8RcYdti/artifacts/user/apideck_task/check.py
import os
import json
import urllib.request

app_id = os.environ.get('APIDECK_APP_ID')
api_key = os.environ.get('APIDECK_API_KEY')
consumer_id = os.environ.get('APIDECK_CONSUMER_ID')

headers = {
    'Authorization': f'Bearer {api_key}',
    'x-apideck-app-id': app_id,
    'x-apideck-consumer-id': consumer_id,
    'x-apideck-service-id': 'onedrive'
}

req = urllib.request.Request('https://unify.apideck.com/file-storage/drives', headers=headers)
with urllib.request.urlopen(req) as response:
    print(json.dumps(json.loads(response.read()), indent=2))
